src/snacks/SnackInfos.stories.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { withInfo } from '@storybook/addon-info'
+import SnackInfos from './SnackInfos'
+
+export default {
+  title: 'SnackInfos',
+  decorators: [withInfo]
```

Comment:
Hast du nicht withInfo schon global hinzugefügt?

Reply:
Nein, ich hatte sie in der config.js für die GlobalStyles gesetzt. Weil die aber keine PropTypes haben, habe ich sie dort wieder entfernt. Jetzt sieht die Info-Seite aufgeräumter aus.